src/style.css

```
@@ -1 +1,12 @@
 /* styles go here */
+fieldset {
+  margin: 20px 20px auto;
```

Why do you need auto here? It's redundant
```suggestion
  margin: 20px;
```